evalml/tests/model_understanding_tests/test_permutation_importance.py

```diff
@@ -473,7 +509,7 @@ def test_get_permutation_importance_multiclass(
         parameters={"Logistic Regression Classifier": {"n_jobs": 1}}, random_seed=42
     )
     pipeline.fit(X, y)
-    for objective in multiclass_core_objectives:
+    for objective in ["Log Loss Multiclass", "AUC Micro"]:
```

Comment:
Would be nice to do something similar to what @chukarsten pointed out here with `multiclass_test_objectives`

Reply:
Done!